app/src/main/res/navigation/my_nav.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<navigation xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/my_nav"
+    app:startDestination="@id/cocktailsFragment">
+
+    <fragment
```

Comment:
Even though Android Studio does not warn us about this, we should set all the labels from the strings.xml file